### PR TITLE
fix: doc SubscriptionConfig

### PR DESCRIPTION
**Pull request by dacleyra**
Document SubscriptionConfig

### CODE REVIEW

File: deployer/v2/config/manifests/bases/CSV_description.md [hunk 1]
```diff
@@ -106,6 +106,13 @@ The operator releases adhere to semantic versioning and provides a seamless upgr
...
 
+### Subscription Config
+
+It is possible to [configure](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md) how OLM deploys an Operator via the `config` field in the [Subscription](https://github.com/operator-framework/olm-book/blob/master/docs/subscriptions.md) object.
+
+The Red Hat Marketplace Deployment Operator will also read the `config` and append it to the operands. The primary use case is to control scheduling using [Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) and [NodeSelectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).
+
+A limitation is that the `config` elements are only appended to the operands. The elements in the operands are not removed if the `config` is removed from the `Subscripion`. The operand must be modify manually, or deleted and recreated by the controller.
```
Comment (mxpaspa) [inline deployer/v2/config/manifests/bases/CSV_description.md:115]: `The operand must be modify manually`
modified*

File: deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -105,7 +105,16 @@ spec:
     Hat Marketplace products, you will need to make the pull secret available across
     the cluster. This can be achieved by applying the Red Hat Marketplace Pull Secret
     as a [global pull secret](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets).
-    For alternative approachs, please see the official OpenShift [documentation](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html).\n\n\n###
+    For alternative approachs, please see the official OpenShift [documentation](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html).\n\n###
```
Comment (mxpaspa) [inline deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml:108]: `For alternative approachs`
approaches*

File: deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -105,7 +105,16 @@ spec:
...
+    Subscription Config\n\nIt is possible to [configure](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md)
+    how OLM deploys an Operator via the `config` field in the [Subscription](https://github.com/operator-framework/olm-book/blob/master/docs/subscriptions.md)
+    object.\n\nThe Red Hat Marketplace Deployment Operator will also read the `config`
+    and append it to the operands. The primary use case is to control scheduling using
+    [Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+    and [NodeSelectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).\n\nA
+    limitation is that the `config` elements are only appended to the operands. The
+    elements in the operands are not removed if the `config` is removed from the `Subscripion`.
+    The operand must be modify manually, or deleted and recreated by the controller.\n\n###
```
Comment (mxpaspa) [inline deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml:117]: `The operand must be modify manually`
modified*

File: v2/config/manifests/bases/ibm-metrics-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -201,8 +201,16 @@ spec:
...
-    \             |Verbs                                     |\n|----------------------|-----------------------|------------------------------------------|\n|apps
+    in a recovery plan.\n\n### Subscription Config\n\nIt is possible to [configure](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md)
+    how OLM deploys an Operator via the `config` field in the [Subscription](https://github.com/operator-framework/olm-book/blob/master/docs/subscriptions.md)
+    object.\n\nThe IBM Metrics Operator will also read the `config` and append it
+    to the operands. The primary use case is to control scheduling using [Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+    and [NodeSelectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).\n\nA
+    limitation is that the `config` elements are only appended to the operands. The
+    elements in the operands are not removed if the `config` is removed from the `Subscripion`.
+    The operand must be modify manually, or deleted and recreated by the controller.\n\n###
```
Comment (mxpaspa) [inline v2/config/manifests/bases/ibm-metrics-operator.clusterserviceversion.yaml:211]: `The operand must be modify manually`
modified

File: deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (dacleyra) [reply deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml]: corrected

File: v2/config/manifests/bases/ibm-metrics-operator.clusterserviceversion.yaml (hunk 4 above)
Comment (dacleyra) [reply v2/config/manifests/bases/ibm-metrics-operator.clusterserviceversion.yaml]: corrected

File: deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (dacleyra) [reply deployer/v2/config/manifests/bases/redhat-marketplace-operator.clusterserviceversion.yaml]: corrected

**Review by mxpaspa — APPROVED** [review]

**Review by jiangchen2019 — APPROVED** [review]

**Pull request merged**